src/manifold/src/constructors.cpp

```diff
@@ -216,6 +215,37 @@ Manifold Manifold::Sphere(float radius, int circularSegments) {
   return Manifold(pImpl_);
 }
 
+/**
+ * Constructs a manifold from lists of vertices and triangles.
+ *
+ * @param vertices A vector of glm::vec3.
+ * @param triangles A vector of glm::ivec3.
+ *
+ */
+Manifold Manifold::CreateFromVertsAndTriangles(
```

Comment:
Thank you for providing benchmarks, but the reason these functions are not as deeply optimized is that they are not generally performance-limiting. So while your point-benchmarks show the differences narrowly, I would be more convinced by a realistic end-to-end test - look at our Samples tests for instance. Where will you be calling so many constructors and so few Boolean ops that the constructors actually dominate? I am honestly interested, as you may well have a use case I haven't considered.

Reply:
@briansturgill This is the most important comment to address.